README.md

```diff
@@ -20,7 +20,7 @@ Is where we do the things.
 1. Is Issue/Pull on Project Board
```

Comment:
Should we add something here after our conversation with @Oxpray-Org/another-api-created-team?